### PR TITLE
Frontbranch3.0

### PR DESCRIPTION
**Pull request by carmen279**

### CODE REVIEW

File: blog/frontend/UnitTesting.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Beginning to test with Jest
+In our fourth week of our path, we learned the basics of testing our Vue components as well as some code smells to avoid.
+
+## Unit testing using Jest
+### Development of unit tests using Jest for Vue
+
+*10-04-2022*
+
+#### Participants & Content Deliverers  <!-- (Do not change this line!!!) -->
```
Comment (miguelliar) [inline blog/frontend/UnitTesting.md]: Delete (my fault, sorry)

File: blog/frontend/UnitTesting.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+4. Code coverage metric pros & cons
+
+**** <!-- (Do not change this line!!!) -->
+
+## Testing the Rick and Morty app developed the previous week
+(Autonomous work) <!-- Comment wheter if it is autonomous or group work -->
+This week, our autonomuos work consisted on implementing some unitary tests of the Rick and Morty exercise implemented for the previous week, which was a simple application using an external API.
+Tough the week was shorter because Easter holidays took Thursday and Friday away from us, we still met the objectives.
+
+### [Miguel's Rick & Morty app] (https://github.com/miguelliar/vue-practice)
```
Comment (miguelliar) [inline blog/frontend/UnitTesting.md]: As there is a blank space that does not perform the link to the repo

**Review by miguelliar — CHANGES_REQUESTED** [review]
Change the commented lines and we can perform the pull request

File: blog/frontend/UnitTesting.md (hunk 1 above)
Comment (miguelliar) [reply blog/frontend/UnitTesting.md:9]: Number line (just in case)

Comment (miguelliar) [reply blog/frontend/UnitTesting.md]: *Line number 9

**Comment by carmen279** [comment]
Requested changes done

**Pull request merged**